magenta/models/music_vae/lstm_models.py

```diff
@@ -282,7 +282,7 @@ class BaseLstmDecoder(base_model.BaseDecoder):
       -`_flat_reconstruction_loss`
   """
 
-  def build(self, hparams, output_depth, is_training=False):
+  def build(self, hparams, output_depth, is_training=True):
```

Comment:
What's the intention behind this change in default value?

Reply:
A pylint error that the default did not match the parent class.